src/pages/about.js

```diff
@@ -1,66 +1,118 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
-
+import { graphql } from 'gatsby';
```

Comment:
Why'd you import graphql?

Reply:
That's how we are getting Alex's and Seth's profile picture.